.github/workflows/codeql-analysis.yml

```diff
@@ -2,7 +2,6 @@ name: CodeQL Analysis
 
 on:
   push:
-    branches: [ "**" ]
```

Comment:
:thumbsup:

Reply:
We cannot have branches and branches-ignore.
So, removing branches that will cover the ignore.